uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/zones/OrchestratorZoneControllerMockMvcTests.java

```diff
@@ -255,10 +266,11 @@ void testDeleteZone_Forbidden() throws Exception {
 
     @Test
     void testUpdateZone_MethodNotImplemented() throws Exception {
-        performMockMvcCallAndAssertError(put("/orchestrator/zones").contentType(APPLICATION_JSON).content(
+        performMockMvcCallAndAssertError(put("/orchestrator/zones").contentType(APPLICATION_JSON_VALUE).content(
```

Comment:
I believe this change is not needed.  Request content type should be set to APPLICATION_JSON

Reply:
Done